serenity-core/src/test/groovy/net/serenitybdd/core/photography/integration/WhenAPhotographerTakesLotsOfScreenshots.groovy

```diff
@@ -27,7 +27,9 @@ class WhenAPhotographerTakesLotsOfScreenshots extends Specification {
             Darkroom.waitUntilClose();
             driver.quit()
         then:
-            photos.each { photo -> Files.exists(photo.pathToScreenshot) }
+            photos.findAll {
+                photo -> Files.exists(photo.pathToScreenshot)
+            }.size() == 11
```

Comment:
These only check the size of the photos collection. They should also check that Files.exists() returns true for each of them.